lib/mix/lib/mix/tasks/test.ex

```diff
@@ -384,6 +384,9 @@ defmodule Mix.Tasks.Test do
       {:ok, %{excluded: excluded, failures: failures, total: total}} ->
         cover && cover.()
 
+        Application.fetch_env!(:ex_unit, :after_suite)
+        |> Enum.each(fn fun -> fun.() end)
```

Comment:
If we do the changes I mentioned above, we won't need this change, as we will run this always after run.